packages/rum-core/src/domain/rumEventsCollection/view/trackViews.ts

```diff
@@ -100,6 +101,8 @@ export function trackViews(
       startClocks,
       viewOptions
     )
+    cleanupTasks.push(() => newlyCreatedView.cleanup())
```

Comment:
<!-- Warning represent user-facing problems. If possible, it's great to follow this kind of comment with a suggestion.  -->
**🔨 warning:** ‏this is a memory leak, because `stop()` is never called outside of tests

Reply:
Indeed 🤔
We could probably maintain active views somewhere until they stop tracking metrics and cleanup those active views after each tests.
I'll look into that.